packages/launch/src/hooks/use-title.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { useEntityProp } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { LAUNCH_STORE } from '../stores';
+
+export function useTitle() {
+	const domain = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedDomain() );
+	const { setDomainSearch } = useDispatch( LAUNCH_STORE );
+	const [ title, setTitle ] = useEntityProp( 'root', 'site', 'title' );
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore
+	const { saveEditedEntityRecord } = useDispatch( 'core' );
```

Comment:
TODO: add TypeScript types for `saveEditedEntityRecord ` (could be also solved when removing code duplication for  `hooks` folder

Reply:
This should be fixed in https://github.com/Automattic/wp-calypso/pull/46921